lib/Parse/ParseDecl.cpp

```diff
@@ -7861,7 +7861,9 @@ static bool isAllowedWhenParsingLimitedSyntax(AccessorKind kind, bool forSIL) {
   case AccessorKind::WillSet:
   case AccessorKind::DidSet:
   case AccessorKind::Read:
+  case AccessorKind::Read2:
```

Comment:
Would it make sense to have more descriptive names here than `Read2`? For example, rename the exiting `Read` to `UnderscoreRead` and then use `Read` for `Read2`.

Reply:
I think Read2 and Modify2 are fine placeholders for now until the final names are decided on.